src/go/rpk/pkg/cli/registry/mode/set.go

```diff
@@ -41,13 +42,19 @@ Acceptable mode values:
   - READONLY
   - READWRITE
   - IMPORT
+
+You can use the --force flag to force setting mode to IMPORT when there are
+existing schemas.
 `,
 		Example: `
 Set the global schema registry mode to READONLY
   rpk registry mode set --mode READONLY
 
 Set the schema registry mode to READWRITE in subjects foo and bar
   rpk registry mode set foo bar --mode READWRITE
+
+Set the schema registry mode to IMPORT while force-skipping the emptiness check
```

Comment:
Not familiar with the context - do we explain the emptiness check elsewhere by any chance? If not, a quick description of what it is / why it happens might be good to add